cypress/e2e/homepage_spec/redirect.cy.ts

```diff
@@ -28,4 +49,28 @@ describe("redirect", () => {
     loginPage.ensureLoggedIn();
     cy.url().should("include", "/facility");
   });
+
+  it("Verify redirection of 'Contribute on GitHub' link", () => {
+    loginPage.clickContributeOnGitHub();
```

Comment:
This won't work unless we disable chromeWebSecurity which is only applicable to Chrome anyways, so just verify the link.

```
cy.get(loginPage.contributeOnGitHubLink)
      .should("have.attr", "href", "https://github.com/ohcnetwork")
      .and("have.attr", "target", "_blank")
      .and("have.attr", "rel", "noopener noreferrer");
```